tests/test_file_watcher_integration.py

```diff
@@ -113,7 +117,11 @@ async def test_file_watcher_exit_iterator(tmp_path: pathlib.Path) -> None:
     number_of_writes = 0
     expected_number_of_writes = 3
 
-    file_watcher = FileWatcher(paths=[str(tmp_path)])
+    file_watcher = FileWatcher(
+        paths=[str(tmp_path)],
+        force_polling=True,
+        polling_interval=timedelta(seconds=0.05),
```

Comment:
50ms sounds a bit extreme, but I guess it's fine...

Reply:
I know, the timer of the test is set to 100ms  so it needed a safe value to ensure the changes are picked. We can increase the timer interval but that'd make the test to take longer